platform/ios/src/MGLMapView.mm

```diff
@@ -288,7 +288,6 @@ @implementation MGLMapView
     /// True if a willChange notification has been issued for shape annotation layers and a didChange notification is pending.
     BOOL _isChangingAnnotationLayers;
     BOOL _isWaitingForRedundantReachableNotification;
-    BOOL _isTargetingInterfaceBuilder;
```

Comment:
I think we should keep this, but set it to `NO` (rather than calling `NSProcessInfo.processInfo.mgl_isInterfaceBuilderDesignablesAgent`) on an #ifdef (like you had previously).

Then you could keep the existing code that uses `_isTargetingInterfaceBuilder`.

Reply:
They main reason to remove this is that `IBDesignable` does provide a little value to `MGLMapView`, the only changes you can see in IB is the ornaments. Removing this code is a sign that we won't support it and we remove the unusable code.

Reply:
Agreed.

We should still periodically check to see if the issue has been fixed on Xcode's side. Perhaps this is better done with a sample project.

Reply:
> the only changes you can see in IB is the ornaments

As illustrated in #929, the main reason we added a designable was because it’s very awkward to work with a custom view in IB: it ends up as an invisible rectangle. We also took advantage of the designable to remind the developer to set the access token; it’s gentler than asserting at launch.

> We should still periodically check to see if the issue has been fixed on Xcode's side. Perhaps this is better done with a sample project.

Yes, especially now that Xcode 10.2 is in beta. Make sure to test both CocoaPods and Carthage. For what it’s worth, in Xcode 10.1 with the iOS navigation SDK installed via Carthage, the designable appears in a split-second, even in a launch storyboard.